connector/src/test/java/it/eng/connector/integration/BaseIntegrationTest.java

```diff
@@ -57,8 +58,12 @@
 public class BaseIntegrationTest {
 
     // starts a mongodb and s3 simulated cloud storage container; the containers are shared among all tests; docker must be running
-    protected static final MongoDBContainer mongoDBContainer = new MongoDBContainer("mongo:7.0.12");
-    protected static final MinIOContainer minIOContainer = new MinIOContainer("minio/minio");
+    protected static final MongoDBContainer mongoDBContainer =
+            new MongoDBContainer(DockerImageName.parse("mongo:7.0.12"))
+                    .withReuse(false);
+    protected static final MinIOContainer minIOContainer =
+            new MinIOContainer(DockerImageName.parse("minio/minio"))
+                    .withReuse(false);
```

Comment:
Should we change the tag to the one we are using in production? minio/minio:RELEASE.2025-04-22T22-12-26Z